travertino/tests/test_node.py

```diff
@@ -431,47 +439,49 @@ def assign_new_applicator_none():
 
 
 def test_assign_style_with_applicator():
-    """Assigning a new style triggers a reapply if an applicator is already present."""
+    """Assigning a new style triggers an apply if an applicator is already present."""
     style_1 = Style(int_prop=5)
     node = Node(style=style_1, applicator=Mock())
 
-    node.style.reapply.reset_mock()
     style_2 = Style(int_prop=10)
     node.style = style_2
 
     # Style copies on assignment.
     assert isinstance(node.style, Style)
     assert node.style == style_2
+    assert node.style.int_prop == 10
     assert node.style is not style_2
 
     assert node.style != style_1
 
+    # call("int_prop") is called when the style object is created.
     # Since an applicator has already been assigned, assigning style applies the style.
-    node.style.reapply.assert_called_once()
+    node.style.apply.mock_calls = [call("int_prop"), call()]
```

Comment:
```suggestion
    assert node.style.apply.mock_calls == [call("int_prop"), call()]
```

Again, I'm not completely sure I follow where the second call to `apply()` is coming from.

Reply:
Same as above, except that in this case the node has already been initialized and a new style is now being assigned to it. Since the node has an applicator, the style should call its `apply()` method (which previously would have been `reapply()`).